src/components/POIForm.component.js

```diff
@@ -187,4 +170,46 @@ class POIForm extends Component {
   }
 }
 
+function FieldGroup({
+  controlId, //indentifier
```

Comment:
A code style thing, but let's be consistent with comments going the line(s) above the referenced code. And have a space after the `//`.

Also did you mean "identifier"?